src/app/services/templates.service.ts

```diff
@@ -21,6 +21,10 @@ import { catchError, map, mergeAll, mergeMap, toArray } from 'rxjs/operators';
 import { PluginDescription, QhanaPlugin } from './plugins.service';
 import { QhanaBackendService } from './qhana-backend.service';
 
+/*
+ * These two interfaces hold the information of templates.
+ */
```

Comment:
Use actual documentation comments (same as javadoc: `/**`) to write documentation comments for the individual interfaces (not groups of interfaces). Also be a bit more specific than "holds the information of templates". (which kind of information? the complete or only partial information? who condumes this information/for whom is the information held?; these could all be things that are helpful documenting/differentiating the interfaes)